modules/images/webp-uploads/load.php

```diff
@@ -451,7 +451,9 @@ function webp_uploads_img_tag_update_mime_type( $image, $context, $attachment_id
 	 *
 	 * @param bool $prefer_smaller_image_file Whether to prefer the smaller image file.
 	 */
-	if ( apply_filters( 'webp_uploads_prefer_smaller_image_file', false ) ) {
+	$prefer_smaller_image_file = apply_filters( 'webp_uploads_prefer_smaller_image_file', false );
+
+	if ( true === $prefer_smaller_image_file ) {
```

Comment:
Personally I like this style of using strict comparison however I have seen a common pattern for WordPress developers to just pass any value that evaluates to `true` and such I think just with checking that the value is not falsy would be enough.

```suggestion
	if ( $prefer_smaller_image_file ) {
```